### ISSUE TITLE
Question more than issue...

### ISSUE THREAD

**Issue by chemitaxis** [treygriffith/short-mongo-id#2]
Can I use it in the opposite way? For example, if I have "iTxuMF", that returns "507f191e810c19729de860ea", thanks!!

**Comment by winduptoy** [comment]
@chemitaxis No. It states in the documentation that it's a one-way function.

**Comment by cacharrin** [comment]
Then? what is used for?

**Comment by flou-ainan** [comment]
The answer for your question is on the readme file bro

`From the README file`
>This is, unfortunately, a **one-way function**. It will reliably produce the same short id for the same MongoDB Id, but the operation **can't be reversed** (it is missing information about the machine id, process id, and most of the counter).
